spec/actors/hyrax/actors/generic_work_actor_spec.rb

```diff
@@ -242,6 +236,7 @@
         expect(old_parent.reload.members).to eq []
       end
     end
+
```

Comment:
@jcoyne I don't know if it is the jet lag or what, but this line is **added** by me, not deleted.  We're in agreement here.